tests/cc/test_cond_gate.cq

```diff
@@ -0,0 +1,96 @@
+# File:         test_cond_gate.cq
+# Purpose:      test cQasm conditional gates with CC backend
+# Notes:
+
+version 1.2
+qubits 17
+
+pragma @ql.name("test_cond_gate")
+pragma @ql.platform("config_cc_s17_direct_iq_openql_0_10.json")
+
+
+.condGate
+measure_fb q[0:1]  # write associated bit registers
+
+#cond (false) rx180 q[0]  # FIXME: vanishes
+cond (true) rx180 q[0]
+
+
+cond (b[1]) rx180 q[0]
+cond (!b[1]) rx180 q[0]
+
+cond (b[0] && b[1]) rx180 q[0]
+cond (b[0] || b[1]) rx180 q[0]
+cond (b[0] ^^ b[1]) rx180 q[0]
+
+# FIXME: seems to break CI, see https://github.com/QuTech-Delft/OpenQL/runs/5588352276?check_suite_focus=true
+# cond (rnd(0.5)) rx180 q[0]
+
+
+.condGateWithInstrDecomposition
+#cond (false) x q[0]
+cond (true) x q[0]
+
+cond (b[1]) x q[0]
+cond (!b[1]) x q[0]
+
+cond (b[0] && b[1]) x q[0]
+cond (b[0] || b[1]) x q[0]
+cond (b[0] ^^ b[1]) x q[0]
+
+# FIXME: seems to break CI, see https://github.com/QuTech-Delft/OpenQL/runs/5588352276?check_suite_focus=true
+# cond (rnd(0.5)) x q[0]
```

Comment:
What does rnd(0.5) do, rounding or random?
Failing on random is ok because it is not predictable.

Reply:
This is for future testing, and is likely to change